cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/core/web/MessageBundleProperties.java

```diff
@@ -0,0 +1,61 @@
+package org.apereo.cas.configuration.model.core.web;
+
+/**
+ * Configuration properties class for message.bundle.
+ *
+ * @author Dmitriy Kopylenko
+ * @since 5.0.0
+ */
+
+public class MessageBundleProperties {
+
+    private String encoding = "UTF-8";
+
+    private int cacheSeconds = 180;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['180' is a magic number.](https://www.codacy.com/app/mmoayyed/cas/file/2769871180/issues/source?bid=3256115&fileBranchId=3357337#l14)